### PR TITLE
Mu 15

### PR DESCRIPTION
**Pull request by Staroz**
Please, review this PR and give me your feedback. Thanks.

### CODE REVIEW

File: cypress/e2e/testBoards.cy.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /// <reference types= "cypress" />
 
-describe("Boards", function() {
+describe("Manipulating UI of Trello", function() {
```
Comment (sphinix27) [inline cypress/e2e/testBoards.cy.js]: I believe you shouldn't change the description of the test

File: cypress/e2e/testBoards.cy.js [hunk 2]
```diff
@@ -9,25 +9,48 @@ describe("Boards", function() {
...
 	});
 
-	describe("Create", function () {
-		it("Create a new board", function () {
+    after(function() {
+        cy.visit('/u/'+this.credentials.userName +'/boards');
+        cy.logout();
+    });
+
+    describe("Create a work space whit UI", function () {
```
Comment (sphinix27) [inline cypress/e2e/testBoards.cy.js]: This test should go in another test file, only for Workspace.

File: cypress/e2e/testBoards.cy.js [hunk 3]
```diff
@@ -9,25 +9,48 @@ describe("Boards", function() {
...
+		it("Create WS", function () {
```
Comment (sphinix27) [inline cypress/e2e/testBoards.cy.js]: Use full word for this test: `Create WorkSpace`

File: cypress/support/commands.js [hunk 4]
```diff
@@ -32,18 +14,27 @@ Cypress.Commands.add('login', (email, pw, userName) => {
...
-    cy.get('[class="frrHNIWnTojsww GDunJzzgFqQY_3 bxgKMAm3lq5BpA HAVwIqCeMHpVKh SEj5vUdI3VvxDc"]').click({force: true});
-    cy.get('[class="board-menu-navigation-item-link js-open-more"]').click();
-    cy.get('[class="board-menu-navigation-item-link js-close-board"]').click({force: true});
-    cy.get('[class="js-confirm full nch-button nch-button--danger"]').click(); 
-    cy.get('[data-testid="close-board-delete-board-button"]').click();
-    cy.get('[data-testid="close-board-delete-board-confirm-button"]').click();
+// HANDLING WORKSPACES AND BOARD WITH UI.
+Cypress.Commands.add('createWorkSpace', (workSpaceName)=> {
+    // I did triple-click for a bug in this button.
+    cy.get('[data-testid="header-create-menu-button"]').dblclick().click();
```
Comment (sphinix27) [inline cypress/support/commands.js:26]: Is it a bug, or maybe another element is being clicked, you can try the option: `{force: true}`

File: cypress/support/commands.js [hunk 5]
```diff
@@ -100,23 +99,65 @@ Cypress.Commands.add('boardDeleteApi', function(key, token) {
...
+            method: 'POST',
+            }).then(function(response){
+                expect(response.status).to.eq(200);
+                const workSpaceId = response.body.id;
+                cy.wrap(workSpaceId).as('workSpaceId');
+                });
+    });
+
+    // FUNCTIONS 
+function getId(arr, boardName) {
```
Comment (sphinix27) [inline cypress/support/commands.js]: This util functions should be in another file and just be imported here, this file should only contain cypress Commands.

File: tests/pages/UiHandling.page.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
+
```
Comment (sphinix27) [inline tests/pages/UiHandling.page.js:1]: This changes shouldn't be in this PR, for the moment I'll just ignore them.

**Review by sphinix27 — CHANGES_REQUESTED** [review]
The main change would be to create a new test file for Worskpace tests in UI.

**Comment by Staroz** [comment]
Please, review this commit. Thanks

**Comment by sphinix27** [comment]
Looks good, Please resolve issues

**Pull request merged**